include/fmt/chrono.h

```diff
@@ -290,36 +291,49 @@ inline null<> localtime_s(...) { return null<>(); }
 inline null<> gmtime_r(...) { return null<>(); }
 inline null<> gmtime_s(...) { return null<>(); }
 
+inline const std::locale& get_classic_locale() {
+  static const auto& locale = std::locale::classic();
+  return locale;
+}
+
 template <typename Char>
-inline auto do_write(const std::tm& time, const std::locale& loc, char format,
-                     char modifier) -> std::basic_string<Char> {
-  auto&& os = std::basic_ostringstream<Char>();
+inline void do_write(buffer<Char>& buf, const std::tm& time,
```

Comment:
nit: I think we can drop `do_` here and below because this prefix is only used in cases with identical signatures. Here signature is specific enough.

Reply:
Done